projects/core/src/lib/layers/legend/legend-raster/raster-legend.component.html

```diff
@@ -41,6 +41,7 @@
                     <td class="classes">
                         <div class="icon" [style.background-color]="breakpoint | breakpointToCssStringPipe"></div>
                     </td>
+                    <td class="classes">{{ breakpoint.value }}</td>
```

Comment:
is the value or the class label more important to the user?
The value could e.g. be very large and then the label is very far apart from the color.
Maybe show the label first and then the value in parentheses?
If you like it better the way it is just keep it.

Reply:
well i guess we should try it like this. if we see a classification with a number larger than 10k we can still change it ...